_posts/2025-09-05-sublimite-financiar.md

```
@@ -0,0 +1,34 @@
+
+Se ha implementado la sección `Sublimite a financiar`, esto ayudará a tener visibilidad del límite de exposición sugerido según segmentación de riesgo, el monto dependerá del tramo al que corresponda el deudor.
+
+Puedes verlo disponible en `Matriz de riesgo` en la tarjeta `Línea de crédito` como último ítem.
+
+![img/sublimite_financiar.png](https://cdn.capitalexpress.cl/img/sublimite_financiar.png)
+
+## Sublimite a financiar - Solicitud de línea de crédito
+
+El `Sublimite a financiar` también es encontrado en la `Solicitud de línea de crédito` en la sección de `Deudores` y `Propuesta deudores`, secciones disponibles para Riesgo y miembros del Comité.
+
+#### Te mostramos como encontrar esta sección en la Solicitud de línea de crédito
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)